Readme.md

```diff
@@ -0,0 +1,3 @@
+git checkout webapi\
+cd ChessBoard.Api\
+docker-compose up -d
```

Comment:
please add a shot description what application does and 

- mongo connection string to check
- url on which the api swagger can be reached after starting the application

Reply:
will resolve after next PR